### ISSUE TITLE
bCacheFS background target for Nix Store

### ISSUE THREAD

**Issue by Krutonium** [NixOS/nixpkgs#316826]
This isn't technically a bug but more of a feature request.

I was looking at https://github.com/koverstreet/bcachefs-tools/issues/235 And I noticed that you can set a background target *per folder or file*

Since NixOS has a lot of churn in the `/nix/store`, it can somtimes lead to a situation where performance is temporarily degraded.

I havn't found any way to do this, but I think it would be fantastic to enable, through nix, the ability to do `sudo bcachefs setattr --background_target=ssd /nix/store/` which sets where the files should be stored. This can't be done at the moment, since the store is read only, but it would enable one to ensure high performance access to the nix store on a bCacheFS system.

So basically what I am asking for is a way to do this - Unless there already is one that I've missed.
